docs/source/index.rst

```diff
@@ -1,82 +1,61 @@
- .. AiiDA-VASP documentation master file, created by
-   sphinx-quickstart on Mon Feb 12 09:24:45 2018.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 Welcome to AiiDA-VASP's documentation!
 ======================================
 
 AiiDA-VASP is a plug-in for the workflow management and data provenance tracking framework `AiiDA`_. It provides the classes `AiiDA`_ needs to run simulations using `VASP`_ (Vienna Ab initio Simulation Package). `VASP`_ is a program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles. For detailed documentation on using `VASP`_ take a look in their `VASP wiki`_.
 
 AiiDA-VASP is under active development, check out the `changelog`_.
 
-The plugin is only tested with VASP 5.4.4 and is compatible with Python>=3.6.
-
-.. warning: Please accept that the development of this plugin is a community effort and any error, bug or missing functionality that might appear is the responsibility of the individual user. If you detect something that needs to be improved we highly encourage to open an issue or even better, submit a pull requests, where you try to fix the issue yourself. You can do this on our Github repository. 
+.. warning::
+   Please accept that the development of this plugin is a community effort and any error, bug or missing functionality that might appear is the responsibility of the individual user. If you detect something that needs to be improved we highly encourage to open an issue or even better, submit a pull requests, where you try to fix the issue yourself. You can do this on our Github repository.
```

Comment:
```suggestion
   Please accept that the development of this plugin is a community effort and any error, bug or missing functionality that might appear is the responsibility of the individual user. If you detect something that needs to be improved we highly encourage to `open an issue <https://github.com/aiida-vasp/aiida-vasp/issues>`_ or even better, submit `pull requests <https://github.com/aiida-vasp/aiida-vasp/pulls>`_, where you try to fix the issue by contributing to the project
```

May include the links. We can also say please don't use aiidausers google mailing list as the first try for getting help...

Reply:
> We can also say please don't use aiidausers google mailing list as the first try for getting help

A very good point and something we have had to deal with a few times. I will add this as well when I update it.